crates/storage/db/src/tables/mod.rs

```diff
@@ -254,6 +260,22 @@ dupsort!(
     ( StorageChangeSet ) TransitionIdAddress | [H256] StorageEntry
 );
 
+table!(
+    /// Stores the current state of an [`Account`] indexed with `keccak256(Address)`
+    /// This table is in preparation for merkelization and calculation of state root.
+    /// We are saving whole account data as it is needed for partial update when
+    /// part of storage is changed. Benefit for merkelization is that hashed addresses are sorted.
+    ( HashedAccount ) H256 | Account
+);
+
+dupsort!(
+    /// Stores the current storage values indexed with `keccak256(Address)` and
+    /// hash of storage key `keccak256(key)`.
+    /// This table is in preparation for merkelization and calculation of state root.
+    /// Benefit for merkelization is that hashed addresses/keys are sorted.
```

Comment:
How are they sorted after hashing? A bit confusing comment.

Also 
```suggestion
    /// Benefit for merklization is that hashed addresses/keys are sorted.
```